config/app.default.php

```diff
@@ -178,6 +182,7 @@
             'password' => 'secret',
             'client' => null,
             'tls' => null,
+            'url' => env('EMAILTRANSPORT_DEFAULT_URL', ''),
```

Comment:
I was trying to just uppercase the key and then replace the nesting with an underscore.